pkgs/applications/version-management/git-and-tools/overcommit/default.nix

```diff
@@ -0,0 +1,15 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv {
```

Comment:
This should probably be `bundlerApp`.

Reply:
With a `bundlerApp` the provided hooks won't work because ruby needs to find the overcommit library.